pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -110,6 +111,8 @@ type Alertmanager struct {
 	silencer *silence.Silencer
 	silences *silence.Silences
 
+	muteTimes map[string][]timeinterval.TimeInterval
```

Comment:
I think this would be nicer as then its consistent with the `Config` struct?

```suggestion
	muteTimeIntervals map[string][]timeinterval.TimeInterval
```

Reply:
I wanted to keep the names equal to the ones in upstream Prometheus, so we have an easier time backtracking stuff we took from upstream if we want to eventually use upstream.